package.json

```diff
@@ -57,42 +57,42 @@
   "private": true,
   "overrides": {
     "@kolkov/ngx-gallery": {
-        "@angular/animations": "^17.3.11",
-        "@angular/common": "^17.3.11",
-        "@angular/core": "^17.3.11"
+      "@angular/animations": "^17.3.11",
+      "@angular/common": "^17.3.11",
+      "@angular/core": "^17.3.11"
```

Comment:
NOTE: These indentation changes were made by `npm` itself.  I didn't modify this file manually, but ran an `npm install` to upgrade Cypress.